### ISSUE TITLE
Previous tags of same name beging changed

### ISSUE THREAD

**Issue by dirad** [formulahendry/vscode-auto-rename-tag#567]
While the extension is enabled, 
renaming a tag that has siblings of same name, they get renamed as well.

Furthermore, undoing does not get back to previous state, unless hit several times. [related issue](https://github.com/formulahendry/vscode-auto-rename-tag/issues/565)

**Comment by SimonSiefke** [comment]
Could you a share some example code for the sibling tag issue?

Renaming sibling tags with the same name seems to work for me.
![renaming](https://user-images.githubusercontent.com/23744935/115056944-b7011b80-9ee3-11eb-8e4d-98a19f09ebca.gif)

**Comment by alexNecroJack** [comment]
Yes @SimonSiefke , I have an example (if I get what @dirad means ).
Step 1 - Set out for failure
```HTML
<div>
    <div>_Cursor_here_
    <div></div>
</div>
```
I just wrote my div...
Step 2 - Failing
```HTML
<div>
    <span>_Cursor_here_
    <div></div>
</span>
```
See how the element at the end of my code got renamed...
I wrote a div opening, and before I close it I decide I need a span instead of div => "Bug" appears.
It's not "someones fault", but the plugin makes it easy to open a logic bug... sadly
